Лабораторная работа 2/task_2.py

```diff
@@ -0,0 +1,10 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен
+money_capital = 0
+for _ in range(1, months + 1):
```

Comment:
```suggestion
for _ in range(months):
```
При таком цикле как вы должны изменить тело цикла, чтобы все работало верно.